time_series_transformer/decoder.py

```diff
@@ -34,6 +39,7 @@ class Decoder(nn.Module):
         One of ``'chunk'``, ``'window'`` or ``None``. Default is ``'chunk'``.
     """
 
+    # pylint: disable=too-many-arguments
```

Comment:
We have a .pylintrc file now, no need for in-code comments

Reply:
Awesome! I will pull that then.

Reply:
Already resolved in the lasted pull request draft.